pkg/controller/issueapi/handle_issue_batch.go

```diff
@@ -65,7 +63,7 @@ func (c *Controller) HandleBatchIssue() http.Handler {
 		}
 
 		// Add realm so that metrics are groupable on a per-realm basis.
-		if !realm.AllowBulkUpload {
+		if realm := controller.RealmFromContext(ctx); realm == nil || !realm.AllowBulkUpload {
```

Comment:
What's the situation in which this can happen?

Reply:
there should never be nil here, but now that I've said that we've probably jinxed it.